modules/finality-verifier/src/lib.rs

```diff
@@ -134,24 +126,42 @@ decl_module! {
 			with_transaction(|| {
```

Comment:
Would it be possible to use the `frame_support_procedural::transactional` attribute macro for the call?

Reply:
Just looking at the docs they look reasonably equivalent. I'm not sure if there any subtle differences though

Reply:
Looks like the [`transactional` macro is implemented using `with_transaction`](https://github.com/paritytech/substrate/blob/master/frame/support/procedural/src/transactional.rs#L23), so they're equivalent